webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux.js';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo.json';
+
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '../bar.js';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the current directory:
+import f4 from './baz/index.js';
```

Comment:
The file `index.js` gets special treatment by webpack. When webpack is searching , if it finds a directory that matches the path, it looks for a file named `index.js` (or `index.jsx` or `index.ts` or `index.tsx`) in that directory. If it finds it, it automatically imports that file.

As above, I think there are lint rules that will complain if you import `index.js` in a React app. So remove it throughout this file as well. For directories, don't include a trailing `/`.